<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tour_add_form</class>
 <widget class="QWidget" name="Tour_add_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление тура</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>221</width>
     <height>146</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="tour_ID">
      <property name="text">
       <string>ID тура</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="tour_ID_input"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="tour_location">
      <property name="text">
       <string>Место отдыха</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="tour_location_input"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="tour_time">
      <property name="text">
       <string>Период отдыха</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="tour_time_input"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="tour_cost">
      <property name="text">
       <string>Стоимость</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="tour_cost_input"/>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="addition">
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
